container_deployment_syntax.md

```diff
@@ -0,0 +1,422 @@
+| **Mandatory keys**     | `cpu`, `mem`                                                                      |
+| **Optional keys**      | `state`, `storage`,`upload_speed`, `download_speed`, `upload`, `download`, `temp` |
+
+Container limits
+
+```YAML
+quotas:
+  cpu: 50
+  mem: 2KB
+  state: 64KB
+  storage: 64KB
+  upload_speed: 32Kb
+  download_speed: 32Kb
+  upload: 1GB
+  download: 1GB
+  temp: 32KB
+```
+
+### quotas key-value: `cpu`
+
+|               |                |
+|:--------------|:---------------|
+| **YAML Type** | int or decimal |
+| **Mandatory** | yes            |
+
+CPU limit (percents of CPU time)​.
+
+### quotas key-value: `mem`
+
+|                  |                                     |
+|:-----------------|:------------------------------------|
+| **YAML Type**    | string                              |
+| **Mandatory**    | yes                                 |
+| **Lark grammar** | [positive integer]["KB"|"MB"|"GB"]  |
+
+RAM memory which accessable for container.
+
+### quotas key-value: `state`
+
+|                  |                                     |
+|:-----------------|:------------------------------------|
+| **YAML Type**    | string                              |
+| **Mandatory**    | no                                  |
+| **Lark grammar** | [positive integer]["KB"|"MB"|"GB"]  |
+
+Maximum size of the state file for container.
+
+### quotas key-value: `storage`
+
+|                  |                                     |
+|:-----------------|:------------------------------------|
+| **YAML Type**    | string                              |
+| **Mandatory**    | no                                  |
+| **Lark grammar** | [positive integer]["KB"|"MB"|"GB"]  |
+
+Size of the storage that accessable for container.
+
+### quotas key-value: `upload_speed`
+
+|                  |                                     |
+|:-----------------|:------------------------------------|
+| **YAML Type**    | string                              |
+| **Mandatory**    | no                                  |
+| **Lark grammar** | [positive integer]["KB"|"MB"|"GB"]  |
+
+Max upload speed per second.
+
+### quotas key-value: `download_speed`
+
+|                  |                                     |
+|:-----------------|:------------------------------------|
+| **YAML Type**    | string                              |
+| **Mandatory**    | no                                  |
+| **Lark grammar** | [positive integer]["KB"|"MB"|"GB"]  |
+
+Max download speed per second.
+
+### quotas key-value: `upload`
+
+|                  |                                     |
+|:-----------------|:------------------------------------|
+| **YAML Type**    | string                              |
+| **Mandatory**    | no                                  |
+| **Lark grammar** | [positive integer]["KB"|"MB"|"GB"]  |
+
+Max count of memory that container able to upload per 24h.
```

Comment:
The maximum amount of download internet traffic that container can utilize per 24h.